packages/tools/fetch-tool/src/fluidFetchMessages.ts

```diff
@@ -27,6 +28,13 @@ function filenameFromIndex(index: number): string {
     return index === 0 ? "" : index.toString(); // support old tools...
 }
 
+// current sequence number to fetch from
+let currSeq: number = 1;
+// flag for mismatch between last sequence number read and new one to be read
+let seqNumMismatch = false;
```

Comment:
No need of this global var. It can just be local in below function as per other comments.